as4-client/src/main/java/dk/toldst/eutk/as4client/userinformation/B2BException.java

```diff
@@ -0,0 +1,16 @@
+package dk.toldst.eutk.as4client.userinformation;
+
+/**
+ * Created by IntelliJ IDEA. Please write your own text here.
+ * User: brj@netcompany.com
+ * Date: 2017-01-05
+ */
+public class B2BException extends Exception {
```

Comment:
Lad os kalde den noget andet end B2BException

Reply:
Changed to AS4Exception!